ledger/participant-integration-api/src/main/scala/platform/store/backend/h2/H2Table.scala

```diff
@@ -33,11 +36,20 @@ private[h2] object H2Table {
                 }
                 preparedStatement.addBatch()
               }
-              preparedStatement.executeBatch()
+              retry(10) {
```

Comment:
Despite we never seen the race issue in CI (probably too fast execution makes it very rare), the issue was revealed by the race-test, a simple retry helps.